.env.example

```diff
@@ -2,6 +2,7 @@ AUTH_SECRET=<your-secret>
 NEXTAUTH_URL=http://localhost:3000
 NEXT_PUBLIC_APP_URL=http://localhost:3000
 NEXT_PUBLIC_API_URL=http://localhost:3000/api
+AUTH_TRUST_HOST=http://localhost:3000
```

Comment:
Great that you're looking into stripe already! Let's keep Stripe as a separate PR though, so we can merge this one first 🙂